build/conf.js

```diff
@@ -21,6 +21,7 @@ import path from 'path';
  * Load the i18n and l10n configuration. Used when dashboard is built in production.
  */
 let localization = require('../i18n/locale_conf.json');
+let util = require('gulp-util');
```

Comment:
Import can be used instead here. Require can be used when we need to 'import' some custom file outside of `node_modules`.